Swift/BRCryptoDemo/CoreDemoAppDelegate.swift

```diff
@@ -77,18 +77,18 @@ class CoreDemoAppDelegate: UIResponder, UIApplicationDelegate, UISplitViewContro
             .urls(for: .documentDirectory, in: .userDomainMask)[0]
             .appendingPathComponent("Core").path
 
-        do {
-            if FileManager.default.fileExists(atPath: storagePath) {
-                try FileManager.default.removeItem(atPath: storagePath)
-            }
-
-            try FileManager.default.createDirectory (atPath: storagePath,
-                                                     withIntermediateDirectories: true,
-                                                     attributes: nil)
-        }
-        catch let error as NSError {
-            print("Error: \(error.localizedDescription)")
-        }
+//        do {
```

Comment:
Is this intended to be committed?

Reply:
Yes.  But it changes in another, future commit.